main/client.go

```diff
@@ -15,9 +15,9 @@ import (
 )
 
 const (
-	serviceUrl  = "http://127.0.0.1:8080" // Local
+	// serviceUrl  = "http://127.0.0.1:8080" // Local
 	// serviceUrl  = "http://34.132.121.53:8080" // Google Cloud
```

Comment:
Technically, we can delete this line now.

Reply:
Let's postpone it, the current client is to be redesigned anyway. It should be converted into an integration test that works on an isolated environment with not leaving garbage in existing databases.